wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1296,6 +1297,56 @@ def workflow_status_with_date(workflow_state):
     return _("%(status_display)s %(task_name)s %(started_at)s") % translation_context
 
 
+@register.inclusion_tag(
+    "wagtailadmin/shared/keyboard_shortcuts_dialog.html", takes_context=True
+)
+def keyboard_shortcuts_dialog(context):
+    user_agent = context["request"].headers.get("User-Agent", "")
+    is_mac = re.search(r"Mac|iPod|iPhone|iPad", user_agent)
+    modifier = "⌘" if is_mac else "Ctrl"
+    shortcuts = {
+        ("common-actions", _("Common Actions")): [
+            {"label": _("Copy"), "shortcut": f"{modifier} + c"},
+            {"label": _("Cut"), "shortcut": f"{modifier} + x"},
+            {"label": _("Paste"), "shortcut": f"{modifier} + v"},
+            {
+                "label": _("Paste without formatting"),
+                "shortcut": f"{modifier} + shift + v",
+            },
+            {"label": _("Undo"), "shortcut": f"{modifier} + z"},
+            {"label": _("Redo"), "shortcut": f"{modifier} + y"},
+        ],
+        ("page-editing-actions", _("Page editing actions")): [
+            {"label": _("Save"), "shortcut": f"{modifier} + s"},
```

Comment:
I didn't know about `⌘ + s` :clap:

Models (snippets) are saved, but pages `Save draft`. Maybe the label should be `Save (draft)`?

This shortcut overrides the default browser behaviour. And in this specific case, that seems okay. Saving HTML is not a common thing to do. Overriding OS/browser shortcuts is something we should consider when adding more shortcuts.

This shortcut doesn't work when the cursor is inside a text-area. It will use the default, trigger the browser to save the HTML. -- Fixing this might be outside the scope of this PR.

Reply:
Great feedback - thanks!

The behaviour for keyboard shortcuts to not listen inside input fields is intentional. However, I see how this could be confusing for the CMD + S scenario. 

I'll look at that for a different PR.

Generally I agree that we should not override keyboard shortcuts in the browser but this has been in the code for a long time. Be sure to have a read of https://github.com/wagtail/wagtail/issues/3949 for the big picture plan for keyboard shortcuts, I've already raised the same concerns there.